.devcontainer/dev-container-setup.sh

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright contributors to the Galasa project
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+function check_required_environemnt_variables () {
+    REQUIRED_ENVS=("GALASA_API_SERVER_URL")
+
+    for VAR in "${REQUIRED_ENVS[@]}"; do
+        if [ -z "${!VAR}" ]; then
+            echo "Error: Required environment variable $VAR is not set! Please set the variable on your host machine and restart VS Code."
```

Comment:
Someone can set the webui's environment variables via a .env.development.local file, which gets loaded when the webui starts up, so I don't think this script is actually needed.

Someone can also just set the environment variable in the terminal within the devcontainer, so you don't have to set it in your host machine.